packages/base/room.gts

```diff
@@ -246,6 +247,14 @@ class PatchField extends FieldDef {
   @field payload = contains(PatchObjectField);
 }
 
+// A map from card document hash to the first card fragment event id.
+// This map can be used to avoid sending the same version of the card more than once in a conversation.
+// We can reuse exisiting eventId if user attached the same version of the card.
+const cardDocToEventId: Map<string, string> = new Map();
+export function retrieveEventId(cardDoc: LooseSingleCardDocument) {
```

Comment:
since this is new API for this module (meant only to be called externally), gets be a bit more precise about the name of the function. how about renaming to `getEventIdForCard()`